<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerIconSwitchModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerIconSwitchModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IconSwitch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="lightButton">
     <property name="text">
      <string>Change to Light</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="darkButton">
     <property name="text">
      <string>Change to Dark</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="iconDisplayFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
